<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionTab</class>
 <widget class="QWidget" name="QuestionTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="lay" stretch="1,0">
   <item>
    <layout class="QHBoxLayout" name="question_layout">
     <item>
      <layout class="QVBoxLayout" name="data_layout">
       <item>
        <widget class="QGroupBox" name="video_group">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QWidget" name="video_label" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="image_group">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="image_label">
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="text_group">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="text_label">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="answer_group">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="answer_button">
          <property name="styleSheet">
           <string notr="true">color: rgb(38, 0, 61);
font: 75 12pt &quot;Levenim MT&quot;;
background-color: rgb(234, 245, 237);
border-radius: 2px;
border: 2px solid #26003d;
box-shadow: 4px 4px 4px rgba(8, 8, 8, 0.5)}
QPushButton:hover {background-color: rgba(200, 229, 225, 0.7); 
border: 2px solid #26003d;
</string>
          </property>
          <property name="text">
           <string>answer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="answer_button_correct">
          <property name="styleSheet">
           <string notr="true">color: rgb(38, 0, 61);
font: 75 12pt &quot;Levenim MT&quot;;
background-color: green;
border-radius: 2px;
border: 2px solid #26003d;
box-shadow: 4px 4px 4px rgba(8, 8, 8, 0.5)}
QPushButton:hover {background-color: rgba(200, 229, 225, 0.7); 
border: 2px solid #26003d;
</string>
          </property>
          <property name="text">
           <string>correct answer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="answer_button_incorrect">
          <property name="styleSheet">
           <string notr="true">color: rgb(38, 0, 61);
font: 75 12pt &quot;Levenim MT&quot;;
background-color: red;
border-radius: 2px;
border: 2px solid #26003d;
box-shadow: 4px 4px 4px rgba(8, 8, 8, 0.5)}
QPushButton:hover {background-color: rgba(200, 229, 225, 0.7); 
border: 2px solid #26003d;
</string>
          </property>
          <property name="text">
           <string>incorrect answer</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttons_layout">
     <item>
      <widget class="QPushButton" name="end_button">
       <property name="styleSheet">
        <string notr="true">color: rgb(38, 0, 61);
font: 75 12pt &quot;Levenim MT&quot;;
background-color: rgb(234, 245, 237);
border-radius: 2px;
border: 2px solid #26003d;
box-shadow: 4px 4px 4px rgba(8, 8, 8, 0.5)}
QPushButton:hover {background-color: rgba(200, 229, 225, 0.7); 
border: 2px solid #26003d;
</string>
       </property>
       <property name="text">
        <string>Завершить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="skip_button">
       <property name="styleSheet">
        <string notr="true">color: rgb(38, 0, 61);
font: 75 12pt &quot;Levenim MT&quot;;
background-color: rgb(234, 245, 237);
border-radius: 2px;
border: 2px solid #26003d;
box-shadow: 4px 4px 4px rgba(8, 8, 8, 0.5)}
QPushButton:hover {background-color: rgba(200, 229, 225, 0.7); 
border: 2px solid #26003d;
</string>
       </property>
       <property name="text">
        <string>Пропустить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="next_button">
       <property name="styleSheet">
        <string notr="true">color: rgb(38, 0, 61);
font: 75 12pt &quot;Levenim MT&quot;;
background-color: rgb(234, 245, 237);
border-radius: 2px;
border: 2px solid #26003d;
box-shadow: 4px 4px 4px rgba(8, 8, 8, 0.5)}
QPushButton:hover {background-color: rgba(200, 229, 225, 0.7); 
border: 2px solid #26003d;
</string>
       </property>
       <property name="text">
        <string>Далее</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
